packages/@react-aria/interactions/test/useFocusVisible.test.js

```diff
@@ -284,6 +295,23 @@ describe('useFocusVisible', function () {
       toggleBrowserWindow();
       expect(el.textContent).toBe('example');
     });
+
+    it('correctly shifts focus to the iframe when the iframe is focused', async function () {
+      ReactDOMRender(<ButtonExample id="iframe-example" />, iframeRoot);
+
+      // Fire focus in iframe
+      await waitFor(() => {
+        expect(document.querySelector('iframe').contentWindow.document.body.querySelector('button[id="iframe-example"]')).toBeTruthy();
+      });
+      addWindowFocusTracking(iframeRoot);
+
+      const el = document.querySelector('iframe').contentWindow.document.body.querySelector('button[id="iframe-example"]');
+
+      fireEvent.focus(iframe.contentWindow.document.body);
+      fireEvent.keyDown(el, {key: 'Tab'});
+
+      expect(el.textContent).toBe('example-focusVisible');
+    });
```

Comment:
@snowystinger I am not able to make this test work. Any idea what I might be doing wrong?